templates/conf.d/nginx.conf.erb

```diff
@@ -326,5 +326,19 @@ stream {
 <% unless @confd_only -%>
   include <%= @conf_dir %>/streams-enabled/*;
 <% end -%>
+
+<% if @stream_log_format -%>
```

Comment:
can you test if the if statement is required? The variable always exists, by default it's just empty. I think we can directly start with the .sort_by call

Reply:
What could make sense is
```ruby
<% end -%>
<% unless @stream_log_format.empty? -%>

<% @stream_log_format.sort_by{|k,v| k}.each do |key,value| -%> 
```

With this you can prevent the empty lines in the generated config. Because right now it generates (roughly):
```
  include .../streams-enabled/*;


  access_log ...
```